src/tiled/tilesetview.cpp

```diff
@@ -241,6 +241,279 @@ static QTransform tilesetGridTransform(const Tileset &tileset, QPoint tileCenter
     return transform;
 }
 
+static void setWangStyle(QPainter *painter,int index, int max)
+{
+    QColor c;
+
+    if (index == 0)
+        c = QColor(0,0,0,0);
+    float hue = ((float)index-1)/(float)max;
+    c = QColor::fromHsvF(hue,1,1);
```

Comment:
Need the `if (index == 0)` part to make alpha zero. But otherwise I cleaned up the casts.